CI/CD implementation

## Tasks 
- [x] #12 
- [x] Implement build stage
- [ ] Implement test stage
- [ ] Implement deploy stage

According to the [Bigtop CI Setup Guide](https://cwiki.apache.org/confluence/display/BIGTOP/Bigtop+CI+Setup+Guide), was added matrix `Bigtop-3.2.1-packages`, but execution returns an error:
```
FAILURE: Build failed with an exception.

* What went wrong:
The specified project directory '/ws/kg' does not exist.

BUILD FAILED in 1s
```

--- a/provisioner/docker/docker-compose.yml
+++ b/provisioner/docker/docker-compose.yml
@@ -25,4 +25,9 @@ services:
         - ./config/hiera.yaml:/etc/puppet/hiera.yaml
         - ./config/hieradata:/etc/puppet/hieradata
         - ./config/hosts:/etc/hosts
-        - /sys/fs/cgroup:/sys/fs/cgroup:ro
+
+networks:
+    default:
+        driver: bridge
+        driver_opts:
+            com.docker.network.driver.mtu: 1450

